h2o-py/setup.py

```diff
@@ -64,7 +64,7 @@
         "Topic :: Scientific/Engineering :: Information Analysis",
 
         # Pick your license as you wish (should match "license" above)
-        "License :: OSI Approved :: Apache Software License",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
```

Comment:
v3.0?

Reply:
Nope, due to pypi check: https://autopilot-docs.readthedocs.io/en/latest/license_list.html